### PR TITLE
generate user, infection and organism models

### PR DESCRIPTION
**Pull request by KseniiaNovinskaia**

### CODE REVIEW

**Comment by KseniiaNovinskaia** [comment]
@Kimon-Haars can you have a look?

File: app/models/infection.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Infection < ApplicationRecord
+  belongs_to :user
+  belongs_to :organism
```
Comment (Kimon-Haars) [inline app/models/infection.rb:3]: why is this not :through?

Comment (KseniiaNovinskaia) [reply app/models/infection.rb:3]: you have access to it through user:id and organism:id of the infections

**Comment by Kimon-Haars** [comment]
Fantastic job Kseniia!!! You are a champion!!!!

**Comment by Kimon-Haars** [comment]
sry.....

**Comment by Kimon-Haars** [comment]
looks good!

**Pull request merged**